pkg/autodiscover/autodiscover_operators.go

```diff
@@ -68,6 +69,19 @@ func findOperatorsByLabel(olmClient clientOlm.Interface, labels []configuration.
 
 	return csvs
 }
+func getAllStorageData(oc corev1client.CoreV1Interface) (data []string, err error) {
```

Comment:
Move this function to https://github.com/test-network-function/cnf-certification-test/blob/main/pkg/autodiscover/autodiscover_pv.go.

This will help us organize the storage related funcs.